docs/developer/release/1-create-release-branch.md

```diff
@@ -0,0 +1,37 @@
+# Create Release Branch
+
+A single release branch is created for every major or minor release. That release
+branch is then used for all Release Candidates, the Stable Release, and all
+Patch Releases.
+
+## Release Version Prefix
```

Comment:
IMO we should be talking about this in terms of semantic versioning rather than the first/second/third numbers, but I think this section is OK for now.